include/dlaf/lapack_tile.tpp

```diff
@@ -10,6 +10,17 @@
 
 #include "dlaf/common/assert.h"
 
+template <class T, Device device>
+void hegst(int itype, blas::Uplo uplo, const Tile<T, device>& a, const Tile<T, device>& b) {
+  DLAF_ASSERT(a.size().rows() == a.size().cols(), "Error: HEGST: matrix A is not square.");
+
+  // itype = 1 to solve inv(U**H)*A*inv(U) or inv(L)*A*inv(L**H))
+  // itype = 2, 3 to solve U*A*U**H or L**H*A*L
+  auto info = lapack::hegst(itype, uplo, a.size().cols(), a.ptr(), a.ld(), b.ptr(), b.ld());
+
+  DLAF_ASSERT(info == 0, "");
```

Comment:
Instead of an empty message, I would report the `info` parameter, otherwise we don't have any information on fault.